pom.xml

```
@@ -62,9 +62,9 @@
       <version>2.4.2</version>
```

The incompatibility justifies a major version increment.
```suggestion
      <version>2.5.0</version>
```